Source/RapyutaSimulationPlugins/Private/Drives/RRJointComponent.cpp

```diff
@@ -33,8 +33,18 @@ void URRJointComponent::SetVelocityTarget(const FVector& InLinearVelocity, const
 
 bool URRJointComponent::HasReachedVelocityTarget(const float InLinearTolerance, const float InAngularTolerance)
 {
-    return LinearVelocityTarget.Equals(LinearVelocity, InLinearTolerance) &&
-           AngularVelocityTarget.Equals(AngularVelocity, InAngularTolerance);
+    float linearTolerance = InLinearTolerance;
+    if (linearTolerance < 0)
+    {
+        linearTolerance = LinearVelocityTolerance;
+    }
+    float angularTolerance = InAngularTolerance;
+    if (angularTolerance < 0)
+    {
+        angularTolerance = AngularVelocityTolerance;
+    }
```

Comment:
```suggestion
    const float linearTolerance = (InLinearTolerance >= 0.f) ? InLinearTolerance : LinearVelocityTolerance;
    const float angularTolerance = (InAngularTolerance >= 0.f) ? InAngularTolerance : AngularVelocityTolerance;
```
to avoid one possible extra re-assignment